services/iam/drivers/IamAccount.py

```diff
@@ -298,3 +304,24 @@
             self.results['PartialEnableConfigService'] = [-1, ', '.join(badResults)]
         else:
             return
+
+    def _checkSCPEnabled(self):
+        # Run this check only when AWS Organization is activated in the account
+        if self.organizationIsEnabled == True:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)